src/Equinox.MessageDb/MessageDb.fs

```diff
@@ -180,85 +67,62 @@ module Read =
     let inline len (bytes: EventBody) = bytes.Length
     let private resolvedEventLen (x: ITimelineEvent<EventBody>) = len x.Data + len x.Meta
     let private resolvedEventBytes events = events |> Array.sumBy resolvedEventLen
-    let private loggedReadSlice reader streamName batchSize requiresLeader startPos batchIndex (log: ILogger) ct: Task<_> = task {
+    let private loggedReadSlice reader streamName batchSize requiresLeader startPos batchIndex ct: Task<_> = task {
         let act = Activity.Current
         let! t, slice = readSliceAsync reader streamName batchSize startPos requiresLeader |> Stopwatch.time ct
         let bytes, count = slice.Messages |> resolvedEventBytes, slice.Messages.Length
-        let reqMetric: Log.Measurement = { stream = streamName; interval = t; bytes = bytes; count = count}
-        let evt = Log.Slice reqMetric
         if act <> null then act.IncMetric(count, bytes).AddLastVersion(slice.LastVersion) |> ignore
-        let log = if not (log.IsEnabled Events.LogEventLevel.Debug) then log else log |> Log.propResolvedEvents "Json" slice.Messages
```

Comment:
what would someone do if they wanted to see the raw json in a debugger?

Reply:
Not sure how this is being used presently. I copied it in from SqlStreamStore, but have never actually ran with debug logs even in dev. When I want to see raw stuff I just run a query against the DB
```sql
select * from read_stream_messages('MyCategory-streamId')
```

Reply:
sometimes selectively upping logging level can be useful for hard to diagnose things
or in messy test scenarios with asynchrony that dont play well with a debugger
or if you want to show the workings in interactive mode
talking about being able to double check encodings and/or escaping etc
For when your json lib converts strings to dates and the like ;)
The MemoryStoreLogger (some is in Propulsion.MemoryStore; some is in the templates) kinda covers this but from a different angle (I'll often turn up the logging for a property test failure where I have the seed)